web-ui/pages/mint-nft/index.vue

```diff
@@ -0,0 +1,386 @@
+      <hr>
+    </section>
+    <section v-if="step === 2">
+      <h2>2. Create NFT Class</h2>
+      <div>
+        <label>Max number of supply for this NFT Class (optional):</label>
+        <input v-model="classMaxSupply" type="number">
+        <p><label>Upload NFT Class data json (nft_class.json) file: </label></p>
+        <input type="file" @change="onClassFileChange">
+        <br>
+        <button :disabled="isLoading" @click="onClassFileInput">
+          Create
+        </button>
+      </div>
+    </section>
+    <section v-else-if="step > 2">
+      <h3>NFT Class Information</h3>
+      <p>
+        NFT Class ID:
+        <a
+          target="_blank"
+          :href="`${likerLandURL}/nft/class/${encodeURIComponent(classId)}`"
+        >
+          {{ classId }}
+        </a>
+      </p>
+      <hr>
+    </section>
+    <section v-if="step === 3">
+      <h2>3. Mint ISCN</h2>
+      <div>
+        <label>Number of NFT to mint:</label>
+        <input v-model="nftMintCount" type="number">
+        <p><label>Upload NFT default data json (nfts_default.json) file: </label></p>
+        <input type="file" @change="onMintNFTDefaultFileChange">
+        <p><label>Upload NFT CSV (nfts.csv) file: </label></p>
+        <input type="file" @change="onMintNFTFileChange">
+        <br>
+        <button :disabled="isLoading" @click="onMintNFTStart">
+          Create
+        </button>
+      </div>
+    </section>
+
+    <section v-if="step > 3">
+      Success!
+      <button :disabled="isLoading" @click="onDownloadCSV">
+        Download NFT result csv
+      </button>
+      <p>
+        <a
+          target="_blank"
+          :href="`${likerLandURL}/nft/class/${encodeURIComponent(classId)}`"
+        >
+          View your NFT
+        </a>
+      </p>
+    </section>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { storeToRefs } from 'pinia'
+import { v4 as uuidv4 } from 'uuid'
+import { parse } from 'csv-parse/sync'
+import { stringify } from 'csv-stringify/sync'
+
+import { useWalletStore } from '~/stores/wallet'
+import { LCD_URL, APP_LIKE_CO_URL, LIKER_LAND_URL } from '~/constant'
+
+const store = useWalletStore()
+const { wallet, signer } = storeToRefs(store)
+const { connect } = store
+
+const appLikeCoURL = APP_LIKE_CO_URL
+const likerLandURL = LIKER_LAND_URL
+const step = ref(1)
+const error = ref('')
+const isLoading = ref(false)
+
+const iscnIdInput = ref('')
+const iscnOwner = ref('')
+const iscnCreateData = ref<any>(null)
+const iscnData = ref<any>(null)
+
+const classData = ref<any>(null)
+const classMaxSupply = ref(0)
+const classCreateData = ref<any>(null)
+
+const nftMintListData = ref<any>([])
+const nftMintDefaultData = ref<any>(null)
+const nftMintCount = ref(100)
+const nftData = ref<any>(null)
+const nftCSVData = ref('')
+
+const iscnId = computed(() => iscnData.value?.['@id'])
+const classId = computed(() => classData.value?.id)
+
+onMounted(async () => {
+
+})
```

Comment:
What is this for?

Reply:
Placeholder that wasn't used, will remove